examples/tutorials/advanced/websockets-example-MNIST/start_websocket_servers.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import sys
```

Comment:
There is also another file in ```webosckets-mnist```? From a first look, they seem identical? Can't one be removed

Reply:
Even we merge this file and make a new file in the main repo. then there has to be a file in `websockets-example-MNIST` and `websockets_mnist` which call this `start_websocket_servers.py` in the main repo which calls `run_websocket_server.py`. So I think there will be a lot of arguments that will involve which will not make codebase shorter but also make it complex.

Reply:
So I think we have to leave it.

Reply:
Why not use directly the file from ```websockets_mnist``` (like call only that file?)

Reply:
So you are saying for `websockets-example-MNIST/start_websocket_servers.py` we can use  `websockets_mnist/start_websocket_servers.py` ?